### ISSUE TITLE
Error when parsing data with Buffer

### ISSUE THREAD

**Issue by ymc9** [blitz-js/superjson#210]
Hi there,

is Buffer type supported? It seems I can stringify but can't parse it back:

```js
const superjson = require('superjson');
superjson.stringify({d: Buffer.from('hello')});
```

Uncaught Error: Trying to deserialize unknown typed array

**Comment by Skn0tt** [comment]
Yup, custom type needed. It's not universally available on all platforms (only Node.js), so we can't make it a default type 🤷
